api/deploy_deployer.go

```diff
@@ -62,6 +63,13 @@ func (d *serviceDeployer) FromGitURL(url string) (*service.Service, *importer.Va
 	if err := xgit.Clone(url, path); err != nil {
 		return nil, nil, err
 	}
+
+	// XXX: remove .git folder from repo.
+	// It makes docker build iamge id same between repo clones.
+	if err := os.RemoveAll(filepath.Join(path, ".git")); err != nil {
```

Comment:
maybe we can have a slice const at the top to specify folders that needs to be deleted for different version control systems.

Reply:
We only support git right now, discussion about different system etc should be continued in another  issue